instrumentation/apache-camel-2.20/javaagent/build.gradle.kts

```diff
@@ -47,21 +54,9 @@ dependencies {
 }
 
 tasks {
-
   withType<Test>().configureEach {
     // TODO run tests both with and without experimental span attributes
     jvmArgs("-Dotel.instrumentation.apache-camel.experimental-span-attributes=true")
     jvmArgs("-Dotel.instrumentation.aws-sdk.experimental-span-attributes=true")
   }
-
-  named<Javadoc>("javadoc") {
```

Comment:
This was ported over from the Groovy before

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/v1.3.1/instrumentation/apache-camel-2.20/javaagent/apache-camel-2.20-javaagent.gradle#L55

But was originally incorrect because dependencies is on `project`, having it under `javadoc` didn't do anything, but apparently it could still work in some cases based on the evaluation order of the project based on other configuration. This PR seems to have triggered a subtle change in order that didn't evaluate `javadoc` in time